js/counter.js

```diff
@@ -31,6 +32,20 @@ class Player {
   save() {
     Cookies.set("player", this)
   }
+  reset() {
+    Cookies.remove("player")
+  }
```

Comment:
I think you could probably have `reset()` _also_ clear the stats. So after line 36, add `this.clearStats()`.

That way, you wouldn't have to call both methods to fully refresh a player. You'd just call `reset()` and it would do both.